lua/telescope/utils.lua

```diff
@@ -284,4 +284,42 @@ function utils.strcharpart(str, nchar, charlen)
   return str:sub(nbyte + 1, nbyte + len)
 end
 
+function utils.get_visual_selection_range()
```

Comment:
I think `:help vim.region()` may be of use to you here. I'm not 100% sure though without reading some more code.

Reply:
Thanks for the pointer, I'll look into it!